debian/ralph-core.templates

```diff
@@ -6,7 +6,7 @@ Description: Use default database settings:
 
 Template: ralphsrv/database_engine
 Type: select
-Choices: mysql, postgresql
+Choices: mysql, postgresql, mariadb
```

Comment:
there are references for ```ralphsrv/database_engine``` in ```ralph-core.config``` and ```ralph-core.postinst```. It is necessary to make appropriate changes at least in ```ralph-core.config```

Reply:
should be resolved.